.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: test-e2e
+
+on:
+  workflow_call:
+    secrets:
+      KNUU_KUBECONFIG_FILE:
+        required: true
+
+jobs:
+  test-e2e:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
I fixed the tags issue by adding `fetch-depth: 0` @rootulp

Reply:
thanks! Looks like that will fetch all tags so SGTM